e3sm_to_cmip/default.py

```diff
@@ -42,7 +42,7 @@ def write_data(varid, data, timeval=None, timebnds=None, index=None, **kwargs):
             else:
                 outdata = data[RAW_VARIABLES[0]].values
 
-        outdata[np.isnan(outdata)] = 1e20 
+        outdata[np.isnan(outdata)] = 1.e20 
```

Comment:
I think you missed this one

Reply:
Thank you, Tom. I left it out intentionally, because it is not one of the individual handlers. We can fix this in next refactor.